### PR TITLE
INDY-903: Removal of client codebase

### PR DESCRIPTION
**Pull request by ArtObr**
Signed-off-by: ArtObr <artemobruchnikov@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,15 +69,10 @@ Indy Node repo consists of the following parts:
       So, if you want to work with Indy Node, you will need to have the Plenum code as well in most of the cases
       and work with two projects at the same time 
       (see [How to Start Working with the Code](#how-to-start-working-with-the-code) below).
```
Comment (ashcherbakov) [inline README.md]: Please edit `dev-setup` scripts and how-to doc to get rid of anoncreds and charm-crypto installation.
The following is no longer needed:
```
echo 'Installing Charm Crypto...'
sudo apt-get install -y python3-charm-crypto
echo 'Installed Charm Crypto'
```

File: indy_client/client/wallet/wallet.py [hunk 2]
```diff
@@ -1,33 +1,27 @@
 import datetime
```
Comment (ashcherbakov) [inline indy_client/client/wallet/wallet.py]: Why can't we just remove this class? Where is it used?

Comment (ArtObr) [reply indy_client/client/wallet/wallet.py]: This class is used in plenum.server.node.Node#wallet. We use it for signing update messages in indy_node.server.node.Node#notify_upgrade_start

Comment (ArtObr) [reply indy_client/client/wallet/wallet.py]: Removed

File: README.md (hunk 1 above)
Comment (ArtObr) [reply README.md]: Fixed

**Pull request merged**